Source/WebKit/WebProcess/Extensions/WebExtensionContextProxy.h

```diff
@@ -72,7 +74,8 @@ class WebExtensionContextProxy final : public RefCounted<WebExtensionContextProx
     URL m_baseURL;
     String m_uniqueIdentifier;
     RetainPtr<NSDictionary> m_manifest;
-    double m_manifestVersion;
+    double m_manifestVersion = 0;
```

Comment:
We use the `{ 0 }` style instead of the `= 0` style. Arbitrary but it’s a style choice.